Scenarios/ALKS_Scenario_4.2_2_PartiallyBlockingTarget_TEMPLATE.xosc

```diff
@@ -1,14 +1,45 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <OpenSCENARIO>
-  <FileHeader revMajor="1" revMinor="1" date="2021-06-18T10:00:00" description="ALKS Scenario 4.2_2 PartiallyBlockingTarget Template" author="BMW AG" />
+  <FileHeader revMajor="1" revMinor="1" date="2021-07-09T10:00:00" description="ALKS Scenario 4.2_2 PartiallyBlockingTarget Template" author="BMW AG" />
   <ParameterDeclarations>
     <!--The ParameterDeclarations section is needed for easy variation.-->
-    <ParameterDeclaration name="Ego_InitPosition_LaneId" parameterType="string" value="-4" />
-    <ParameterDeclaration name="Ego_InitSpeed_Ve0" parameterType="double" value="16.66666667" />
-    <ParameterDeclaration name="Duration" parameterType="double" value="40" />
-    <ParameterDeclaration name="TargetBlocking_Catalog" parameterType="string" value="PedestrianCatalog" />
-    <ParameterDeclaration name="TargetBlocking_Model" parameterType="string" value="pedestrian" />
-    <ParameterDeclaration name="TargetBlocking_InitPosition_Offset" parameterType="double" value="-1.5" />
+    <ParameterDeclaration name="Ego_InitPosition_LaneId" parameterType="string" value="-4">
+      <!--A positive lane number would position the entities on the left side of the road. For this the road needs to be adjusted to left hand traffic (attribute "rule"="LHT").-->
+      <ConstraintGroup>
+        <ValueConstraint rule="lessOrEqual" value="-3" />
+        <ValueConstraint rule="greaterOrEqual" value="-5" />
+      </ConstraintGroup>
+      <ConstraintGroup>
+        <ValueConstraint rule="greaterOrEqual" value="3" />
+        <ValueConstraint rule="lessOrEqual" value="5" />
+      </ConstraintGroup>
+    </ParameterDeclaration>
+    <ParameterDeclaration name="Ego_InitSpeed_Ve0_kph" parameterType="double" value="60.0">
+      <ConstraintGroup>
+        <!--The scenarios are meant to be run with a positive ego speed up to 60 kph.-->
+        <ValueConstraint rule="greaterThan" value="0.0" />
+        <ValueConstraint rule="lessOrEqual" value="60.0" />
+      </ConstraintGroup>
+    </ParameterDeclaration>
+    <ParameterDeclaration name="TargetBlocking_Catalog" parameterType="string" value="PedestrianCatalog">
+      <!--Not constrained so it's possible to use different catalogs.-->
+    </ParameterDeclaration>
+    <ParameterDeclaration name="TargetBlocking_Model" parameterType="string" value="pedestrian">
+      <!--Not constrained so it's possible to use different models.-->
+    </ParameterDeclaration>
+    <ParameterDeclaration name="TargetBlocking_InitPosition_LateralOffset_m" parameterType="double" value="-1.5">

```

Comment:
Consider include this into fully blocking as variation of offset is possible

Reply:
discussed and agreed to keep the scenarios separately.